store/config.go

```diff
@@ -0,0 +1,141 @@
+package store
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/pkg/elasticetcd"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pelletier/go-toml"
```

Comment:
We don't have a dependency on direct dependency on yaml. We have an indirect on through spf13/viper.

I prefer toml to yaml. Mainly because toml ignores indentation, and yaml doesn't. This yaml feature has left me frustrated many times.